stylesheets/components/form/_checkbox.styl

```diff
@@ -71,5 +71,9 @@
     text-transform: uppercase
     width: calc(100% - 42px)
 
+    &--serif
```

Comment:
Talked to Sebastian, and all radios and checkboxes should use Lora, so this can go and updates made to the main classes.